features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/App.js

```diff
@@ -43,7 +43,7 @@ class Protected extends Component {
         /* TODO: need to fix the header to avoid conflicting with messages ~tmkb*/
         this.handleResponse = this.handleResponse.bind(this);
         /* TODO: Get apim base url from config*/
-        Axios.get("https://localhost:9292/login/login/store").then(this.handleResponse);
+        Axios.get(location.origin + "/login/login/store").then(this.handleResponse);
```

Comment:
Created issue https://github.com/wso2/carbon-apimgt/issues/5087 to remove all occurrences of hard coded app contexts (/publisher and /store).